packages/fabric/README.md

```diff
@@ -0,0 +1,30 @@
+# Fabric Plugin
+
+## Overview
+Fabric and the Subscription Token Protocol extends the Non Fungible Token standard with the addition of a temporal dimension, and economic incentives for distribution, adoption, growth, and retention.
+
+Documentation: https://docs.withfabric.xyz/stp/overview
+
+Mint Gallery: https://hypersub.withfabric.xyz/
+
+### Mint Plugin
+
+The mint plugin targets the mint functions on the [SubscriptionTokenV1](https://basescan.org/address/0x90e1c94ca5e161acb420e59d8be5041db931c9dd#code) contract. There are 4 functions that mints may be routed through.
+
+- `mint`
+- `mintFor`
+- `mintWithReferral`
+- `mintWithReferralFor`
+
+The `mintFor` methods are used when the NFT is gifted to another address. Referral methods are used when referral metadata is attached to the transaction. Otherwise these all are very similar.

```

Comment:
When we implement the simulation and intent I think this is the function we should hit here. Like the EOA support, that'll be huge when we start using abstracting accounts more.